inc/osvr/ClientKit/Imaging_decl.h

```diff
@@ -50,23 +49,19 @@ namespace clientkit {
     /// @brief The user-friendly imaging report. Note that passing this around
     /// by value is OK (doesn't copy the image), and the easiest way to ensure
     /// that the image buffer doesn't get freed before you're done using it.
-    struct ImagingReportOpenCV {
+    struct ImagingReport {
         /// @brief The device sensor number this frame came from.
         OSVR_ChannelCount sensor;
 
-        /// @brief An OpenCV Mat header for accessing the image data in the
-        /// buffer. Does not store the data internally (to reduce copies)!
-        cv::Mat frame;
-
```

Comment:
This is the main piece that distinguishes the C++ wrapper from the C API. I suppose the lifetime management could be separated from the use of OpenCV, but either way the changes to any C++ `.h` files in this directory do not actually affect the build or runtime requirements of a client that does not explicitly use this file by choice.